src/features/classification/security-controls/utils.js

```diff
@@ -99,20 +96,9 @@ const getSharedLinkMessages = (controls: Controls): Array<MessageItem> => {
 const getWatermarkingMessages = (controls: Controls): Array<MessageItem> => {
     const items = [];
     const isWatermarkEnabled = getProp(controls, `${WATERMARK}.enabled`, false);
+
     if (isWatermarkEnabled) {
```

Comment:
qq: (completely okay if we want to do this in another PR) but since we're here, could we clean up the watermarking check since it's fully GA? thank you!

Reply:
I'll file a ticket for this since it's already a bit outside of the scope of what I'm currently working on